NEWS.md

```diff
@@ -96,6 +96,8 @@
 
 14. `setNumericRounding()` now invisibly returns the old rounding value instead of `NULL`, which is now consistent with similar behavior by `setwd()`, `options()`, etc. Thanks @MichaelChirico for the report and @joshhwuu for the fix.
 
+15. Extended support for non-standard evaluation (NSE) constructs within `[` and `melt()` functions. Now, constructs like `.`, `J`, `measure`, and `patterns` are exported, ensuring they are consistently available. Previously, only `.N`, `.I`, `.GRP`, `.GRPI`, `.SD`, and `:=` were exported. This update helps maintain consistency and functionality across your data.table usage, [#5604](https://github.com/Rdatatable/data.table/issues/5604) and [#5277](https://github.com/Rdatatable/data.table/issues/5277). Thanks @tdhock for suggestions and @Nj221102 for implementing this improvement.
```

Comment:
What is "Extended support" ?? can this first sentence be deleted? If not, why not? What value does it add?
"constructs like" implies to me that there may be others that are exported, that are not listed here, so can that be deleted too please? ( to avoid confusion) 
Also, news items typically start with the function/object name, can you please edit? How about `.`, `J`, `measure`, and `patterns` are exported...

Reply:
done 👍 , if needed further changes plz suggest.